sdk/src/trace/span.cc

```diff
@@ -94,6 +94,10 @@ Span::~Span()
 void Span::SetAttribute(nostd::string_view key, const common::AttributeValue &value) noexcept
 {
   std::lock_guard<std::mutex> lock_guard{mu_};
+  if (recordable_ == nullptr)
+  {
```

Comment:
Do we need emit a log message in this case (`recordable_ == nullptr`)?

Reply:
We don't emit in the other methods.